{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [
          {
            "children": [],
            "cpe_match": [
              {
                "cpe23Uri": "cpe:2.3:a:ibm:entirex:11.1:*:*:*:*:*:*:*",
                "cpe_name": [],
                "vulnerable": true
              }
            ],
            "operator": "OR"
          },
          {
            "children": [],
            "cpe_match": [
              {
                "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:-:*:*:*:*:*:*:*",
                "cpe_name": [],
                "vulnerable": false
              },
              {
                "cpe23Uri": "cpe:2.3:o:microsoft:windows:-:*:*:*:*:*:*:*",
                "cpe_name": [],
                "vulnerable": false
              }
            ],
            "operator": "OR"
          }
        ],
        "cpe_match": [],
        "operator": "AND"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "psirt@us.ibm.com",
      "ID": "CVE-2025-0158"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM EntireX 11.1 could allow a local user to cause a denial of service due to an unhandled error and fault isolation."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://www.ibm.com/support/pages/node/7182693",
          "refsource": "",
          "tags": [
            "Vendor Advisory"
          ],
          "url": "https://www.ibm.com/support/pages/node/7182693"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2025-07-08T20:06Z",
  "publishedDate": "2025-02-06T21:15Z"
}